reactioncommerce/reaction
# Fix Order shipped email total undefined

Resolves #3120 
This PR resolve Order shipped total price is undefined to Float. The cause of the issue was that prices was Stored as Strings which causes a NaN error. Before summing, the prices have to be cast to a Float to convert them to Numbers.

#### Test
* Login as Admin.
* Place an order and checkout.
* Check the email received to see the total price shown.
